### ISSUE TITLE
Error running Supabase with a local Postgres instance with custom SQL ( self-host docker )

### ISSUE THREAD

**Issue by Nirajkashyap** [supabase/supabase#28515]
# Bug report

- [x] I confirm this is a bug with Supabase, not with my own application. 
- [x] I confirm I have searched the [Docs](https://docs.supabase.com), GitHub [Discussions](https://github.com/supabase/supabase/discussions), and [Discord](https://discord.supabase.com). 

## Describe the bug
 
I read the install instructions on https://supabase.com/docs/guides/self-hosting/docker from top to bottom but am unable to launch supabase on my server.
 
i want to add custom sql in supabase. i have placed custom sql script in db directory ( https://github.com/supabase/supabase/tree/master/docker/volumes/db )
![Screenshot 2024-08-09 at 8 32 12 PM](https://github.com/user-attachments/assets/19a7925a-049a-4682-b1f1-e2c1c6fbf14f)
  
i have added custom sql in docker-compose.yml file. 
![Screenshot 2024-08-09 at 8 26 16 PM](https://github.com/user-attachments/assets/b48d081c-a603-4a8f-9088-809b8c8230c0)

i have also disabled analytics and commented depends on analytics where ever analytics is used. example : 
![Screenshot 2024-08-09 at 9 23 56 PM](https://github.com/user-attachments/assets/3195f206-739f-4a16-9652-be5d8172a405)

below is custom SQL ( authTable.sql ) : 
 
![image](https://github.com/user-attachments/assets/92079b5e-3eda-4a8a-a7bc-7dba9091613b)
     

## To Reproduce

Steps to reproduce the behavior, please provide code snippets or a repository:

1. Clone https://github.com/supabase/supabase
2. Go to docker dir 
3. run `docker-compose -f docker-compose.yml up -d `
4. See error docker containers are unhealthy 

## Expected behavior

all docker containers should be healthy and supabase should be working fine for self host with docker. 

## Screenshots
below are error screenshots. 

![Screenshot 2024-08-09 at 9 27 28 PM](https://github.com/user-attachments/assets/bd538f3a-d820-4c4b-bc7d-c96433b06050)

## System information

- OS: macOS ( any docker environment )

## Additional context

similar pr where i have added custom SQL. https://github.com/supabase/supabase/pull/28514

**Comment by Nirajkashyap** [comment]
below error in supabase auth container :

`{"level":"fatal","msg":"running db migrations: error executing /usr/local/etc/auth/migrations/20210909172000_create_identities_table.up.sql, sql: -- adds identities table \n\nCREATE TABLE IF NOT EXISTS auth.identities (\n    id text NOT NULL,\n    user_id uuid NOT NULL,\n    identity_data JSONB NOT NULL,\n    provider text NOT NULL,\n    last_sign_in_at timestamptz NULL,\n    created_at timestamptz NULL,\n    updated_at timestamptz NULL,\n    CONSTRAINT identities_pkey PRIMARY KEY (provider, id),\n    CONSTRAINT identities_user_id_fkey FOREIGN KEY (user_id) REFERENCES auth.users(id) ON DELETE CASCADE\n);\nCOMMENT ON TABLE auth.identities is 'Auth: Stores identities associated to a user.';\n: ERROR: cannot display a value of type pg_ddl_command (SQLSTATE 0A000)","time":"2024-08-09T16:16:14Z"}
`

**Comment by encima** [comment]
Hi @Nirajkashyap 
Thanks for opening. What you are trying to do is not something we support in any official way.

To debug the issue, are you able to run the SQL directly on your custom database with no issues?

**Comment by Nirajkashyap** [comment]
@encima Yes i am able to run the SQL directly in postgres database with superadmin rights.
